src/runtime/spirv/spirv_shader.h

```diff
@@ -31,7 +31,7 @@ namespace tvm {
 namespace runtime {
 namespace vulkan {
```

Comment:
I suppose it should be another namespace.
```suggestion
namespace spirv {
```